packages/elements/src/components/ui/buttons/menu-button/MenuButtonContent.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { Box, Indent, Row, Text } from "@stenajs-webui/core";
+import { Icon } from "../../icon/Icon";
+import styles from "./MenuButton.module.css";
+import { IconDefinition } from "@fortawesome/fontawesome-svg-core";
+
+export interface MenuButtonContentProps {
+  leftIcon?: IconDefinition;
+  label: string;
+}
+
+export const MenuButtonContent: React.FC<MenuButtonContentProps> = ({
+  label,
+  leftIcon,
+}) => {
+  return (
+    <Row>
+      {leftIcon && (
+        <>
+          <Box alignItems={"center"} justifyContent={"center"} width={"16px"}>
+            <Icon
+              icon={leftIcon}
+              size={16}
+              color={"var(--current-text-color)"}
+              data-hover={true}
```

Comment:
What is data-hover? :)

Reply:
Umm, I'm actually unsure. I copied that from Collapsible :D